src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -510,16 +517,39 @@ def to_white_list(
             for idx, host in enumerate(hosts if hosts else default_hosts)
         ]
 
-    def to_ingress(self) -> V1AlphaIngressRoute | None:
-        return (
-            V1AlphaIngressRoute.from_spec(
-                metadata=self._to_object_meta(name="ingress-routes"),
-                spec=self.deployment.traefik.ingress_routes.get_value(self.target),
+    def _replace_placeholders(self, traefik_object: dict | list):
+        for placeholder, replacement in {
+            PR_NUMBER_PLACEHOLDER: str(
+                self.step_input.run_properties.versioning.pr_number
+            ),
+            SERVICE_NAME_PLACEHOLDER: self.release_name,
+            NAMESPACE_PLACEHOLDER: self.namespace,
+            CLUSTER_ENV_PLACEHOLDER: get_cluster_config_for_project(
+                self.step_input.run_properties, self.project
+            ).cluster_env,
+        }.items():
+            traefik_object = replace_item(traefik_object, placeholder, replacement)
```

Comment:
Calling that method 4 times is actually less lines of code 😂 and IMHO simpler to read:
```python
traefik_object = replace_item(traefik_object, PR_NUMBER_PLACEHOLDER, str(self.step_input.run_properties.versioning.pr_number))
traefik_object = replace_item(traefik_object, SERVICE_NAME_PLACEHOLDER, self.release_name)
traefik_object = replace_item(traefik_object, NAMESPACE_PLACEHOLDER, self.namespace)
traefik_object = replace_item(traefik_object, CLUSTER_ENV_PLACEHOLDER, get_cluster_config_for_project(self.step_input.run_properties, self.project).cluster_env)
```